lib/parser.dart

```diff
@@ -80,47 +80,50 @@ class HtmlParser {
 
   final List<ParseError> errors = <ParseError>[];
 
-  String container;
```

Comment:
This looks like a breaking change (it is a public class). Mention it in the changelog?

Reply:
I forgot that I had intended to handle these cleanup breaking changes in separate PRs. Fixed this one in https://github.com/dart-lang/html/pull/150 and included it in the changelog.